src/java.desktop/share/native/libharfbuzz/hb-algs.hh

```diff
@@ -875,7 +879,9 @@ hb_in_ranges (T u, T lo1, T hi1, Ts... ds)
 static inline bool
 hb_unsigned_mul_overflows (unsigned int count, unsigned int size, unsigned *result = nullptr)
 {
-#if (defined(__GNUC__) && (__GNUC__ >= 4)) || (defined(__clang__) && (__clang_major__ >= 8))
+#if defined(AIX)
```

Comment:
The built-in is potentially supported by various compilers on AIX such as GCC or the Open XL C/C++ compilers.